model/log/iterator_interface.go

```diff
@@ -17,3 +22,24 @@ type LogIterator interface {
 	// iterator is no longer needed.
 	Close() error
 }
+
+// newTailIterator converts a log iterator into a basic iterator that reads the
```

Comment:
I think this is ok to keep here because it is a general function for log iterators. Theoretically, it could return any type of iterator. I could even set the return type as `LogIterator` if that would clear things up.